### ISSUE TITLE
Problem with `extract_vector_norm` in and `threshold` in Python 3

### ISSUE THREAD

**Issue by davidcortesortuno** [enthought/mayavi#449]
Hi, I'm trying to filter some data from a VTK file but this fails in Python 3.
I created a minimal working example using one of the Mayavi webpage examples:

In this example, I create an unstructured grid with two cells and some `scalar` and `vector` data on them. Then I just extract the vector norms and try to set a lower threshold of `0.2`  so only one of the cells will be shown, since the other cell has norm equal to 1

```python
from numpy import array, random
from tvtk.api import tvtk
import numpy as np

points = array([[0,1.2,0.6], [1,0,0], [0,1,0], [1,1,1], # tetra
                [1,0,-0.5], [2,0,0], [2,1.5,0], [0,1,0],
                [1,0,0], [1.5,-0.2,1], [1.6,1,1.5], [1,1,1], # Hex
                ], 'f')
# The cells
cells = array([4, 0, 1, 2, 3, # tetra
               8, 4, 5, 6, 7, 8, 9, 10, 11 # hex
               ])
# The offsets for the cells, i.e. the indices where the cells
# start.
offset = array([0, 5])
tetra_type = tvtk.Tetra().cell_type # VTK_TETRA == 10
hex_type = tvtk.Hexahedron().cell_type # VTK_HEXAHEDRON == 12
cell_types = array([tetra_type, hex_type])
# Create the array of cells unambiguously.
cell_array = tvtk.CellArray()
cell_array.set_cells(2, cells)
# Now create the UG.
ug = tvtk.UnstructuredGrid(points=points)
# Now just set the cell types and reuse the ug locations and cells.
ug.set_cells(cell_types, offset, cell_array)

# Data ------------------------------------------------------------------------

scalars = random.random(2)
vectors = np.array([[0, 0, 1.], [0, 0, 0.1]])
ug.cell_data.scalars = scalars
ug.cell_data.scalars.name = 'scalars'
ug.cell_data.vectors = vectors
ug.cell_data.vectors.name = 'vector_data'

# -----------------------------------------------------------------------------


import mayavi.mlab as mlab

f = mlab.figure()

vnorm = mlab.pipeline.extract_vector_norm(ug)
vtres = mlab.pipeline.threshold(vnorm)
vtres.lower_threshold = 0.2
mlab.pipeline.surface(vtres)
mlab.show()

```

This works fine in Python 2 but it doesn't in Python 3, where I get the following Error:

```
Exception
In /home/david/anaconda3/lib/python3.5/site-packages/mayavi-4.5.0-py3.5-linux-x86_64.egg/mayavi/filters/threshold.py:227
AttributeError: 'VectorNorm' object has no attribute 'point_data' (in _get_data_range)
```

This even fails if I try to do a `cell_to_point_data` , so I don't know if it is a bug.

Regards :)

**Comment by prabhuramachandran** [comment]
I think I have fixed this issue in master with PR #425.  Can you please test with master, thanks!  Do let me know if you still have this issue.

**Comment by davidcortesortuno** [comment]
It is working now, using Mayavi from the master branch. Thanks.
